main.go

```diff
@@ -123,6 +119,41 @@ func (m model) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 			m.activeButton = "yes"
 		}
 
+	// Mouse handlers
+	case tea.MouseMsg:
+		if msg.Type == tea.MouseWheelUp {
+			m.list.CursorUp()
+			return m, nil
+		}
+
+		if msg.Type == tea.MouseWheelDown {
+			m.list.CursorDown()
+			return m, nil
+		}
+
+		if msg.Type == tea.MouseRelease {
+			if m.selectedPort == "" {
+				for i, listItem := range m.list.VisibleItems() {
+					item, _ := listItem.(item)
+					// Check each item to see if it's in bounds.
+					if zone.Get(item.title).InBounds(msg) || zone.Get(item.desc).InBounds(msg) {
+						// If so, select it in the list.
+						m.list.Select(i)
+						port := m.list.Items()[i].FilterValue()
+						m.selectedPort = port
+						break
+					}
+				}
+				// If ok is clicked
+			} else if zone.Get("ok").InBounds(msg) {
```

Comment:
With the zones, it'd also be nice to introduce a "hover" state style to show the active button.

Reply:
This was something I looked into for a bit, but not sure zone supports it out of the box. I did not investigate for too long but it didnt look like zone provided anything for hover

It might be nice as another investigation/feature to implement supporting hover, if its possible